helix-query.yaml

```diff
@@ -3,6 +3,10 @@ indices:
   default:
     include:
       - /blogs/**
+      - /blogs/de/**
+      - /blogs/fr/**
+      - /blogs/nl/**
+      - /blogs/uk/**
```

Comment:
locale is first in the url
https://www.servicenow.com/de/blogs.html
https://www.servicenow.com/fr/blogs.html
etc...